cmd/brew-test-bot.rb

```diff
@@ -533,12 +533,17 @@ def download
           "git", "-C", @tap.path.to_s,
                  "log", "-1", "--format=%h (%s)", "origin/master"
         ).strip
-        puts "#{@tap} origin/master #{tap_origin_master_revision}"
         tap_revision = Utils.popen_read(
           "git", "-C", @tap.path.to_s,
                  "log", "-1", "--format=%h (%s)"
         ).strip
-        puts "#{@tap} HEAD #{tap_revision}"
+        puts "\nTesting tap #{@tap} with:"
+        puts "  - origin/master     - #{tap_origin_master_revision}"
+        puts "  - pull request HEAD - #{tap_revision}"
+      else
+        puts "\nTesting Homebrew/brew #{@tap} with commits:"
```

Comment:
`puts "Homebrew/brew #{brew_version} (#{brew_commit_subject})"` covers some of this above.

Reply:
So I've removed the `else`